### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [getAlby - lightning-browser-extension/getAlby - lightning-browser-extension](https://hosted.weblate.org/projects/getalby-lightning-browser-extension/getalby-lightning-browser-extension/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: src/i18n/locales/de/translation.json [hunk 1]
```diff
@@ -386,11 +481,12 @@
...
     },
     "enable": {
-      "allow": "Erlaube du {{host}} zu:",
+      "allow": "Erlauben Sie {{host}} zu:",
       "title": "Verbinden",
       "request1": "Genehmigung für Transaktionen anfordern",
       "request2": "Fordern Sie Rechnungen und Lightning Informationen an",
-      "block_and_ignore": "Blockieren und Ignorieren von {{host}}"
+      "block_and_ignore": "Blockieren und Ignorieren von {{host}}",
+      "block_added": "Hinzugefügt {{host}} zur Blocklist, bitte Website neu laden."
```
Comment (jankoegel) [inline src/i18n/locales/de/translation.json:489]:
```suggestion
      "block_added": "{{host}} wurde zur Blockliste hinzugefügt, bitte Website neu laden."
```

File: src/i18n/locales/de/translation.json [hunk 2]
```diff
@@ -386,11 +481,12 @@
       }
     },
     "enable": {
-      "allow": "Erlaube du {{host}} zu:",
+      "allow": "Erlauben Sie {{host}} zu:",
```
Comment (jankoegel) [inline src/i18n/locales/de/translation.json:484]:
```suggestion
      "allow": "Erlaube {{host}}:",
```

File: src/i18n/locales/de/translation.json [hunk 3]
```diff
@@ -270,7 +270,8 @@
           "failed_to_request_sms": "SMS-Code konnte nicht angefordert werden",
           "failed_to_login_sms": "Anmeldung mit SMS-Code fehlgeschlagen",
           "setup_failed": "Einrichtung fehlgeschlagen",
-          "missing_jwt": "JWT fehlt, Anmeldung nicht möglich."
+          "missing_jwt": "JWT fehlt, Anmeldung nicht möglich.",
+          "invalid_jwt": "ungültige JWT übermittelt"
```
Comment (jankoegel) [inline src/i18n/locales/de/translation.json:274]:
```suggestion
          "invalid_jwt": "ungültigen JWT übermittelt"
```

imo it's DER token.

Comment (bumi) [reply src/i18n/locales/de/translation.json:274]: text is managed in weblate. weblate then opens PRs, but we should edit it in weblate.

Comment (escapedcat) [reply src/i18n/locales/de/translation.json:274]: Currently @jungusemlevve is working on the German translations. Might be good to tag them here to raise awareness.

"Test, test" @jungusemlevve do you get notifications for these comments?

Comment (jungusemlevve) [reply src/i18n/locales/de/translation.json:274]: @escapedcat Hallo, gibt es Unstimmigkeiten?

Comment (escapedcat) [reply src/i18n/locales/de/translation.json:274]: Schau mal ueber @jankoegel 's Kommentare hier bitte. Da Deutsch eh noch nichts live ist, koennen wir dies trotzdem mergen und Du kannst das ja spaeter korriegieren. Oder?

**Review by escapedcat — APPROVED** [review]
Merging this for now. German can be fixed step by step.

**Pull request merged**